contrib/cctz-cmake/CMakeLists.txt

```diff
@@ -614,7 +614,9 @@ if (USE_INTERNAL_CCTZ)
 
         add_library(tzdata STATIC ${TZ_OBJS})
         set_target_properties(tzdata PROPERTIES LINKER_LANGUAGE C)
-        target_link_libraries(cctz -Wl,--whole-archive tzdata -Wl,--no-whole-archive) # whole-archive prevents symbols from being discarded
+        # whole-archive prevents symbols from being discarded
+        # export-dynamic allows to search bundled symbols with dlsym
+        target_link_libraries(cctz -Wl,--push-state,--whole-archive,--export-dynamic tzdata -Wl,--pop-state)
```

Comment:
Maybe use `target_link_options` for this?

Reply:
The only difference is two commands instead of one?)

Reply:
Also, it's not available in CMake less than to 3.13.